src/misc/folderDetails.ts

```diff
@@ -14,18 +15,27 @@ export class FolderDetails {
     public toZip: string;
     public infoFile: string;
     public mainFile: string;
+    public imageIconFile: string;
     public info: IAppInfo;
 
     constructor(private command: Command) {
         this.setFolder(process.cwd());
         this.mainFile = '';
+        this.imageIconFile = '';
         this.info = {} as IAppInfo;
     }
 
     public async doesFileExist(file: string): Promise<boolean> {
         return await fs.pathExists(file) && fs.statSync(file).isFile();
     }
 
+    public doesFolderExist(file: string): boolean {
+        return fs.existsSync(file);
+    }
+
+    public async isFolderEmpty(file: string): Promise<boolean> {
+       return await emptyDir(file);
+    }
```

Comment:
We should probably have a linter rule here, but please add another line between these two functions.

Reply:
Made the change @graywolf336, please see